e2e/test/scenarios/dashboard/x-rays.cy.spec.js

```diff
@@ -280,6 +280,7 @@ describe("scenarios > x-rays", { tags: "@slow" }, () => {
         });
 
         visitDashboardAndCreateTab({ dashboardId: dashboard_id });
+        cy.findByText("There's nothing here, yet.");
```

Comment:
It did not help for me, try running the spec with a "Fast 3G" throttling. There is something else that changes the UI, I'm investigating it